src/net.cpp

```diff
@@ -851,6 +851,12 @@ static bool CompareOnionTimeConnected(const NodeEvictionCandidate& a, const Node
     return a.nTimeConnected > b.nTimeConnected;
 }
 
+static bool CompareI2PTimeConnected(const NodeEvictionCandidate& a, const NodeEvictionCandidate& b)
+{
+    if (a.m_is_i2p != b.m_is_i2p) return b.m_is_i2p;
+    return a.nTimeConnected > b.nTimeConnected;
+}
+
```

Comment:
What would be the purpose of the follow-up you're envisioning ?

AFAIU this current patchset, any unused protected slots across a high-latency network is reallocated to the next one by summing up `protect_size_per_network` and passing through `total_to_protect` ?

Reply:
You're right. The commit message was out of date. Updated.